lambda/src/main/resources/db/migration/V121__update_Name_for_title_alternate_property.sql

```diff
@@ -0,0 +1,9 @@
+-- Update the displayProperties
+
+
+UPDATE "DisplayProperties"
+SET "Value" = 'Add alternative title without the file extension'
```

Comment:
Will this affect the name of the column in the export csv?

Reply:
Yeah, it will. We are using the Name attribute as a column name in the metadata csv.

Reply:
Do you mean the CSV that is generated for the bagit package? If so not sure we can change this

Reply:
No, the CSV file which we download from the "download metadata" or "view transfer" page